libs/common/src/constants/date-timezones.ts

```diff
@@ -0,0 +1,8 @@
+import { IanaTimezone } from '../types';
+
+/**
+ * IANA timezones used in the codebase
+ * Note - If needing a full list, we should look to Intl.supportedValuesOf('timeZone')
+ * {@link https://www.iana.org/time-zones}
+ */
+export const IANA_TIMEZONES = { LONDON: 'Europe/London' } as const satisfies Record<string, IanaTimezone>;
```

Comment:
Can we match the file name with the const please

Reply:
updated -- kept date prefix for ease of finding in file browser